samples/oauth_node/package.json

```diff
@@ -26,7 +26,7 @@
     "openid-client": "^2.5.0",
     "passport": "^0.4.1",
     "pug": "^2.0.4",
-    "sqlite3": "^4.2.0"
+    "sqlite3": "https://github.com/mapbox/node-sqlite3/tarball/master/tree/v4.2.0"
```

Comment:
This is the only one that raises my eyebrows a bit. Why the URL reference here to the same version?

Reply:
Nevermind me. The question is answered in the PR description. But I am still curious about the "why" of not wanting to build locally with `node-gyp`. I can't make sense of that based on the ticket.

Reply:
I was actually not able to get it to build with `node-gyp`, but I can change it back